In [1]:
import pandas as pd
import numpy as np 
import seaborn as sns

In [4]:
karoi = pd.read_csv('/Users/aliably/Desktop/Raw Data /Karoi_E.csv')
karoi

21. What causes climate change? (G, R, D, C, O)  \
0                                                 D   
1                                                 D   
2                                                 G   
3                                                 D   
4                                                 R   
..                                              ...   
126                                               O   
127                                               O   
128                                               R   
129                                               G   
130                                               D   

    22. How did you know about climate change?  (R, N, P, O, A)  \
0                                                    N            
1                                                    P            
2                                                    R            
3                                                    R            
4                                                    N            
..                                                 ...            
126                                                  R            
127                                                  O            
128                                                  O            
129                                                  O            
130                                                  O            

    23. How do you know when to plant crops? (R, E, T, P)  \
0                                                    T      
1                                                    T      
2                                                    R      
3                                                    R      
4                                                  NaN      
..                                                 ...      
126                                                  R      
127                                                  O      
128                                                  E      
129                                                  P      
130                                                  P      

    24. What TBI to predict when to plant crops? (F, B, I, P, O)  \
0                                                 BIPF             
1                                                 BIPF             
2                                                 BIFP             
3                                                 BPIF             
4                                                BIPF              
..                                                 ...             
126                                                 FB             
127                                                  P             
128                                                 IB             
129                                               PFIB             
130                                                  P             

    25. What TBI do you use to predict droughts? (F, B, I, P, O)  \
0                                                 FIPB             
1                                                 BIFP             
2                                                 IPFB             
3                                                 BFIP             
4                                                BIPF              
..                                                 ...             
126                                                  F             
127                                                  P             
128                                                  O             
129                                               BFIP             
130                                                  O             

    26. Who taught you the TBI? (F, P, G, V, O)  \
0                                           NaN   
1                                             G   
2                 

In [5]:
karoi['21. What causes climate change? (G, R, D, C, O)']

0      D
1      D
2      G
3      D
4      R
      ..
126    O
127    O
128    R
129    G
130    D
Name: 21. What causes climate change? (G, R, D, C, O), Length: 131, dtype: object

In [6]:
karoi['21. What causes climate change? (G, R, D, C, O)'].value_counts()

D      45
R      30
O      22
G      18
C      11
D/C     1
D/R     1
Name: 21. What causes climate change? (G, R, D, C, O), dtype: int64

In [7]:
ndambe = pd.read_csv('/Users/aliably/Desktop/Raw Data /Ndambe_CSV.csv')
ndambe

Gender (M/F)  Age (number)  Agriculture experience (number)  \
0             F          56.0                             33.0   
1             F          50.0                             38.0   
2             M          55.0                              NaN   
3             M          28.0                             16.0   
4             F          34.0                             10.0   
5             F          39.0                             20.0   
6             F          34.0                             15.0   
7             F          56.0                             35.0   
8             F          52.0                             35.0   
9             F           NaN                              NaN   
10            F          32.0                             12.0   
11            F          33.0                             13.0   
12            M          28.0                             15.0   
13            F          59.0                             30.0   
14            F          45.0                             15.0   
15            M          59.0                             30.0   
16            F          56.0                             20.0   
17            F          62.0                             35.0   
18            F          56.0                             30.0   
19            F          73.0                             50.0   
20            F          39.0                             20.0   
21            F          78.0                             50.0   
22            F          39.0                             19.0   
23            F          49.0                             27.0   
24            F          42.0                             19.0   
25            M          62.0                             40.0   
26            F          37.0                              NaN   
27            F          70.0                             38.0   
28            F          58.0                             37.0   
29            M          52.0                             15.0   
30            F          62.0                             30.0   
31            F          71.0                             50.0   
32            F          48.0                             30.0   
33            M          85.0                             63.0   
34            M          70.0                             45.0   
35            F          62.0                             30.0   
36            M          65.0                             30.0   
37            F          40.0                             18.0   
38            M          55.0                             27.0   
39            M          58.0                             30.0   
40            F          72.0                             50.0   
41            F          68.0                             38.0   
42            M          52.0                             10.0   
43            F          62.0                             42.0   
44            M          70.0                             42.0   
45            M          73.0                             35.0   
46            F          65.0                             39.0   
47            F          63.0                             30.0   
48            M          69.0                             40.0   
49            F          64.0                             19.0   
50            M          28.0                             15.0   
51            F          46.0                             22.0   
52            F          38.0                             18.0   
53            F          42.0                             20.0   
54            F          35.0                             10.0   
55            F          50.0                             27.0   
56            F          39.0                             20.0   
57            M          28.0                             10.0   

   Religion (C, M, A, X, O, N) Education (N, P, S, A)  \
0                            C                     

In [9]:
ndambe['What causes climate change? (G, R, D, C, O)'].value_counts()

R          42
G          10
O/PLTCS     1
D           1
C           1
O           1
Name: What causes climate change? (G, R, D, C, O), dtype: int64

Karoi: total = 128, 35% deforestation, 23% rejecting culture
Ndambe: total = 56, 75% rejecting culture, 17.85% gods anger
